src/pages/AddBoxForm/AddBoxForm.css

```diff
@@ -0,0 +1,5 @@
+.addBoxForm h1 {
```

Comment:
CSS naming conventions use dashes in between words rather than camel case. For example, this should be named "add-box-form" instead of "addBoxForm". Please change the other CSS files to follow this as well.